slack-api-client/src/main/java/com/slack/api/methods/RequestFormBuilder.java

```diff
@@ -3124,4 +3183,46 @@ private static <T> String getJsonWithGsonAnonymInnerClassHandling(List<T> tList)
         return GSON_ANONYM_INNER_CLASS_INIT_OUTPUT.equals(json) ? GSON.toJson(new ArrayList<>(tList)) : json;
     }
 
+    private static EntityMetadata[] prepareEntities(EntityMetadata[] entities) {
+        List<EntityMetadata> updatedEntities = new ArrayList<EntityMetadata>();
+        for (EntityMetadata entity : entities) {
+            entity = prepareEntityMetadata(entity);
+            updatedEntities.add(entity);
+        }
+        EntityMetadata[] entityArray = new EntityMetadata[updatedEntities.size()];
+        entityArray = updatedEntities.toArray(entityArray);
+
+        return entityArray;
+    }
+
+    private static EntityMetadata prepareEntityMetadata(EntityMetadata entity) {
```

Comment:
```suggestion
    private static EntityMetadata setEntityMetadataFields(EntityMetadata entity) {
```

🏷️ suggestion: Similar thought to avoid "prepare" statements-

Reply:
Updated the naming here as well